src/components/LPolyline.vue

```diff
@@ -0,0 +1,45 @@
+<script>
+import { onMounted, ref, inject } from "vue";
+import { remapEvents, propsBinder } from "../utils.js";
+import { props, setup as polylineSetup } from "../functions/polyline";
+
+/**
+ * Polyline component, lets you add and personalize polylines on the map
+ */
+export default {
+  name: "LPolyline",
+  props,
+  setup(props, context) {
+    const leafletRef = ref({});
+    const ready = ref(false);
+
+    const addLayer = inject("addLayer");
+
+    const { options, methods } = polylineSetup(props, leafletRef, context);
+
+    onMounted(async () => {
+      const { polyline, DomEvent, setOptions } = await import(
+        "leaflet/dist/leaflet-src.esm"
+      );
+
+      leafletRef.value = polyline(props.latLngs, options);
+
+      const listeners = remapEvents(context.attrs);
+      DomEvent.on(leafletRef.value, listeners);
```

Comment:
Should we also be calling `DomEvent.off` on this and other components, when they unmount?

Reply:
@mikeu 🤔  probably yes, I'll open an issue to tackle this on all the components